responsibleai/environments/responsibleai-ubuntu20.04-py38-cpu/context/conda_dependencies.yaml

```diff
@@ -17,10 +17,10 @@ dependencies:
     - pdfkit==1.0.0
     - plotly==5.6.0
     - kaleido==0.2.1
-    - azureml-core=={{latest-pypi-version}}
-    - azureml-dataset-runtime=={{latest-pypi-version}}
-    - azureml-mlflow=={{latest-pypi-version}}
-    - azureml-telemetry=={{latest-pypi-version}}
+    - azureml-core==1.48.0
+    - azureml-dataset-runtime==1.48.0
+    - azureml-mlflow==1.48.0
+    - azureml-telemetry==1.48.0
     - mltable==1.0.0
-    - https://publictestdatasets.blob.core.windows.net/packages/pypi/responsibleai_tabular_automl/responsibleai_tabular_automl-0.0.0-py3-none-any.whl
+    - https://publictestdatasets.blob.core.windows.net/packages/pypi/responsibleai_tabular_automl/responsibleai_tabular_automl-0.1.0-py3-none-any.whl
```

Comment:
These packages should be in PyPI before environments get to azureml-assets.

Reply:
We plan to keep this pypi feed until we complete the integration and then move to a package in sdk-v2 repo with upload to pypi. Is that ok?

Reply:
Created a pypi package for this whl file here:- https://pypi.org/project/responsibleai-tabular-automl/